receiver/mongodbatlasreceiver/testdata/alerts/cert/cert.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please integrate with the [`certs` make target](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/Makefile#L335).